python/cuml/test/test_tsvd.py

```diff
@@ -18,23 +18,46 @@
 from cuml.test.utils import array_equal
 import cudf
 import numpy as np
+import pandas as pd
+from sklearn.utils import check_random_state
+from sklearn.datasets.samples_generator import make_blobs
 
 
 @pytest.mark.parametrize('datatype', [np.float32, np.float64])
 @pytest.mark.parametrize('input_type', ['dataframe', 'ndarray'])
-def test_tsvd_fit(datatype, input_type):
-    X = np.array([[-1, -1], [-2, -1], [-3, -2], [1, 1], [2, 1], [3, 2]],
-                 dtype=datatype)
+@pytest.mark.parametrize('name', [pytest.param('None', marks=pytest.mark.unit),
```

Comment:
Do you want the string 'None' or just `None` here?

Reply:
changed it to `None` in all the places